doc/Quick_start/deployment.md

```diff
@@ -23,9 +41,9 @@
     * More precise explanation of the required fields:
         * For Windows computes use `bms_win` dict instead of regular `bms`
         * Roles supported on Windows are `vrouter` and `win_docker_driver` - specify them
-        * Set `WINDOWS_PHYSICAL_INTERFACE` to dataplane interface alias
-        * Specify `WINDOWS_ENABLE_TEST_SIGNING` if you want to install untrusted artifacts (this is needed at this moment)
-        * Set `WINDOWS_DEBUG_DLLS_PATH` to path on Ansible machine containing MSVC 2015 debug dlls, specifically: `msvcp140d.dll`, `ucrtbased.dll` and `vcruntime140d.dll`
+        * Set `WINDOWS_PHYSICAL_INTERFACE` to dataplane interface name (run `Get-NetAdapter` from PowerShell to list available interfaces on Windows compute node)
+        * Add `WINDOWS_ENABLE_TEST_SIGNING` option and leave it empty, if you want to install untrusted artifacts (this is needed at this moment)
+        * Set `WINDOWS_DEBUG_DLLS_PATH` to path on Ansible machine containing MSVC 2015 debug dlls, specifically: `msvcp140d.dll`, `ucrtbased.dll` and `vcruntime140d.dll`. DLLs can be obtained by installing Windows SDK on Windows and copying DLLs to Ansible machine.
```

Comment:
We should specify that it's for debug build only and not required.

BTW we can't release debug versions of artifacts, can we?

Reply:
We should also say where to put those dlls (which dir) and where to obtain them (you need visual studio...)